continuum/scenarios/huggingface_incremental.py

```diff
@@ -0,0 +1,57 @@
+from typing import List, Union, Tuple
+
+import numpy as np
+import datasets
+from datasets.arrow_dataset import Dataset as HFDataset
+
+from continuum.scenarios import _BaseScenario
+
+
+class HuggingFaceIncremental(_BaseScenario):
```

Comment:
Is it not possible to wrap HuggingFace datasets and using normal ClassIncremental class?